definitions/family-private/json/ComplexTypes.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "10/08/2021",
+    "ID": "DocumentUploaded",
+    "ListElementCode": "DocumentLink",
+    "FieldType": "Document",
+    "ElementLabel": "Document Url",
+    "SecurityClassification": "Public"
+  }
```

Comment:
why have a complex type? is the `Document` type not enough? Or is it to enable future refactoring?

Reply:
I think it's for future needs. I like this approach.

Reply:
It's to use a collection of documents rather than just one.